.nuke/build.schema.json

```diff
@@ -95,6 +98,7 @@
               "BuildLinux",
               "BuildOsx",
               "BuildWindows",
+              "CalculateVersion",
```

Comment:
This wasn't updated in a previous PR, nuke needs to run when the build script changes to pick up the schema change